### PR TITLE
Hotels: Filter stripe refactor + fix scroll to start

### PR DESCRIPTION
**Pull request by tbergq**

### CODE REVIEW

File: app/hotels/src/filter/Filters.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+type Props = {|
+  +children: React.Node,
+|};
+
+export default function Filters(props: Props) {
+  const activeFilters = [];
+  const inActiveFilters = [];
+
+  React.Children.forEach(props.children, child => {
+    if (child.props.isActive) {
```
Comment (mrtnzlml) [inline app/hotels/src/filter/Filters.js]: Nice! Finally, someone figured out how to refactor it... 🙂

File: app/hotels/src/filter/Filters.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+import * as React from 'react';
+
+type Props = {|
+  +children: React.Node,
+|};
+
+export default function Filters(props: Props) {
+  const activeFilters = [];
+  const inActiveFilters = [];
```
Comment (RobinCsl) [inline app/hotels/src/filter/Filters.js]: If I understand correctly, this should read _inactive_ and not _in active_, right? In that case, I would replace `inActive` by `inactive`.

Comment (tbergq) [reply app/hotels/src/filter/Filters.js]: you are of course right, i will change it to lowercase `a`

**Comment by tbergq** [comment]
@RobinCsl grammar fixed

**Pull request merged**